k-distribution/tests/regression-new/checks/invalidConstantExp.k.out

```diff
@@ -0,0 +1,6 @@
+[Error] Compiler: Division by zero.
+  while executing phase "constant expression folding" on sentence at
+	Source(invalidConstantExp.k)
+	Location(4,8,4,21)
+	Source(invalidConstantExp.k)
+	Location(4,13,4,21)
```

Comment:
Why does the error message look weird? It has two sources and location info. And the first two lines are not properly wrapped.
I think the error message is composed of two error messages, but they end up looking inconsistent with the K format and confusing.

Reply:
It's one error message, and this is what it normally looks like when you get an error during a compilation phase. One location is the location of the error itself as reported by the compiler pass, the other error is the location of the rule that was being compiled when the error arose. There are cases, such as with macro expansion, where both of these locations are useful. So I'm not inclined to change this behavior.